opentreemap/exporter/models.py

```diff
@@ -58,3 +61,6 @@ def get_url_if_ready(self):
     def complete_with(self, filename, file_obj):
         self.outfile.save(filename, file_obj)
         self.status = self.COMPLETE
+
+    def fail(self):
+        self.status = self.UNCAUGHT_EXCEPTION_ERROR
```

Comment:
Fair point. I thought about this, but decided to leave it explicit (and repetitive) because this is how the similar method `complete_with` works. Also, I think django model methods that call `save()` and don't have `save` in the name are a little bit dangerous.